### PR TITLE
Improve WebUI Inference Server Configuration Interaction

### PR DESCRIPTION
**Pull request by SapphireLab**
**Is this PR adding new feature or fix a BUG?**

> [!NOTE]
> This PR modified `fish_speech/webui/manage.py`  to improve the user experience when setting the inference models.
> 这一 PR 主要修改了  `fish_speech/webui/manage.py`  以提升设置推理服务器配置时的用户体验.

> [!CAUTION]
> **Fix BUG**
> - The step of `from fish_speech import xxx` must come after `sys.path.insert()` to ensure the successful import.
>
> **修复问题**
> - `from fish_speech import xxx` 这一步应放置在 `sys.path.insert()` 之后以确保正确导入.

> [!TIP]
> **Add Feature** 
> 1. Add warning information in the terminal if there is no available model in local dirs.
> 2. Add `list_llama_model()` to allow users to choose from available LLaMA model paths.
> 3. Add two functions (`change_llama_config`, `change_decoder_config()`) to change the config of the model according to the selection of the model path to ensure the correspondence between models and configurations.
> 4. Change `"Changing with the Model Path"` instead of `"Type the path or select from the dropdown"` and add to i18n's JSON files.
>
> **优化事项**
> 1. 启动 WebUI 时若 `checkpoints` 和 `Results` 路径下没有可用模型时在终端显示警告信息.
> 2. 添加 `list_llama_model()` 函数以识别出可用的 LLaMA 模型路径供用户选择.
> 3. 为解码器和 LLaMA 分别添加修改函数, 用于修改相应配置选项的下拉菜单, 以确保模型和配置相对应. 
(如 `checkpoints\text2semantic-sft-large-v1.1-4k.pth` 需要对应 `dual_ar_2_codebook_large`).
> 4. 修改模型配置下拉菜单的提示为 `随模型路径变化`, 并添加到界面当前支持的语言 (i18n).

**Is this pull request related to any issue? If yes, please link the issue.**

No.

### CODE REVIEW

File: fish_speech/webui/manage.py [hunk 1]
```diff
@@ -16,14 +16,15 @@
 from loguru import logger
 from tqdm import tqdm
 
-from fish_speech.i18n import i18n
-from fish_speech.webui.launch_utils import Seafoam, is_module_installed, versions_html
-
 PYTHON = os.path.join(os.environ.get("PYTHON_FOLDERPATH", ""), "python")
```
Comment (leng-yue) [inline fish_speech/webui/manage.py:19]: We can use the following code instead

```
import pyrootutils
pyrootutils.setup_root(__file__, indicator=".project-root", pythonpath=True)
```

File: fish_speech/webui/manage.py [hunk 2]
```diff
@@ -131,6 +132,34 @@ def change_label(if_label):
         yield build_html_ok_message("Nothing")
 
 
+def change_decoder_config(decoder_model_path):
+    if "vits" in decoder_model_path:
+        choices = ["vits_decoder_finetune", "vits_decoder_pretrain"]
+        return {"__type__": "update", "choices": choices, "value": choices[0]}
```
Comment (leng-yue) [inline fish_speech/webui/manage.py:138]: Returning `gr.Dropdown(choices=choices, value=choices[0])` should be a better solution,

**Review by leng-yue — COMMENTED** [review]
Nice PR, can you also run `python -m fish_speech.i18n.scan` after updating i18n?

File: fish_speech/webui/manage.py (hunk 2 above)
Comment (SapphireLab) [reply fish_speech/webui/manage.py:138]: okay, fix it.

**Comment by SapphireLab** [comment]
> Nice PR, can you also run `python -m fish_speech.i18n.scan` after updating i18n?

I run the command and found two missing keys in `tools/webui.py`  and `webui/manage.py`, so complement.

File: fish_speech/webui/manage.py (hunk 1 above)
Comment (leng-yue) [reply fish_speech/webui/manage.py:19]: 这个感觉改不改都行, 你懒得改我就直接 merge 了 hhh

**Review by AnyaCoder — APPROVED** [review]
OK, nice.

**Pull request merged**